WcaOnRails/app/models/competition.rb

```diff
@@ -164,7 +165,8 @@ def build_clone
 
       Competition.reflections.keys.each do |association_name|
         case association_name
-        when 'registrations', 'results', 'competitors', 'competitor_users', 'delegate_report',
+          # NOTE: 'country' is not cloned because it's obtained through 'countryId' attribute which is clone
+        when 'registrations', 'results', 'country', 'competitors', 'competitor_users', 'delegate_report',
              'competition_delegates', 'competition_events', 'competition_organizers', 'media', 'scrambles'
           # Should be cloned.
```

Comment:
Then it's indented too much, but I'd be up for just using *shouldn't* and it's obvious that the rest *should* be cloned.